src/package.json

```diff
@@ -87,12 +87,12 @@
     "is-empty-object": "^1.1.1",
     "localforage": "^1.5.0",
     "mini-css-extract-plugin": "^0.5.0",
-    "minimongo": "^6.5.0",
+    "minimongo": "^6.8.2",
     "node-sass": "^5.0.0",
     "onchange": "^6.1.0",
     "path": "^0.12.7",
-    "react": "^15.6.2",
-    "react-dom": "^15.6.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-redux": "^5.1.2",
     "react-router": "^3.2.6",
```

Comment:
can you confirm that the non-upgraded versions of react are compatible with v17

Reply:
React router current  version is 3.2.6

React router latest version 6.0.0 depends on React 16.x only. 
ref: https://github.com/ReactTraining/react-router/blob/master/package.json#L39

Current codebase would have to evolve a lot to support  react v17.x and react-dom v17.x
Hence falling back to v15.x